src/main/java/org/folio/circulation/resources/RenewalResource.java

```diff
@@ -74,6 +77,7 @@ private void renew(RoutingContext routingContext) {
       .thenComposeAsync(r -> r.after(requestQueueRepository::get))
       .thenCompose(r -> r.combineAfter(configurationRepository::findTimeZoneConfiguration,
         LoanAndRelatedRecords::withTimeZone))
+      .thenCompose(r -> r.after(overdueFineCalculatorService::calculateOverdueFine))
```

Comment:
Please can all of the permissions needed for all operations that are used in this process be added to the appropriate permission set in the module descriptor

Reply:
Added (in new PR)